quartermaster/src/main/java/com/github/reflectoring/infiniboard/quartermaster/widget/rest/SourceDataResource.java

```diff
@@ -0,0 +1,21 @@
+package com.github.reflectoring.infiniboard.quartermaster.widget.rest;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.springframework.hateoas.ResourceSupport;
+
+import com.github.reflectoring.infiniboard.packrat.source.SourceData;
+
+public class SourceDataResource extends ResourceSupport {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Override this superclass' "equals" method. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS2160)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Document this public class. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AUndocumentedApi)